packages/shared/lib/wallet.ts

```diff
@@ -266,12 +273,10 @@ export async function asyncSyncAccountOffline(account: IAccountState): Promise<v
                 const meta = await getAccountMeta(account.id)
                 const startdustAccount = await get(profileManager).getAccount(account.id)
                 const _account = prepareAccountInfo(startdustAccount, meta)
-                get(activeProfile)?.accounts.update((_accounts) =>
-                    _accounts.map((a) => (a.id === _account.id ? _account : a))
-                )
-                updateActiveProfile({
-                    hiddenAccounts: (get(activeProfile)?.hiddenAccounts || []).filter((id) => id !== _account.id),
-                })
+                activeAccounts.update((_accounts) => _accounts.map((a) => (a.id === _account.id ? _account : a)))
+                // updateActiveProfile({
+                //     hiddenAccounts: (get(activeProfile)?.hiddenAccounts || []).filter((id) => id !== _account.id),
```

Comment:
to include this logic as well, we need to set the `hidden` flag of `_account` to `true`

Reply:
These functions no longer work and are just a reference when we come to right them. So I am not going to modify this.